mne_bids/write.py

```diff
@@ -1352,3 +1353,162 @@ def write_anat(bids_root, subject, t1w, session=None, acquisition=None,
     nib.save(t1w, t1w_basename)
 
     return anat_dir
+
+
+def mark_bad_channels(channels, descriptions=None, *, bids_basename,
+                      bids_root, kind=None, overwrite=False, verbose=True):
+    """Update which channels are marked as "bad" in an existing BIDS dataset.
+
+    Parameters
+    ----------
+    channels : str | list of str | empty list
+        The names of the channel(s) to mark as bad. Pass an empty list in
+        combination with ``overwrite=True`` to mark all channels as good.
+    descriptions : None | str | list of str
+        Descriptions of the reasons that lead to the exclusion of the
+        channel(s). If ``None``, no descriptions are added.
```

Comment:
say that it should match the length of ch_names

Reply:
Fixed in e71f549